docs/PARAMETER.md

```diff
@@ -30,6 +30,28 @@ When a variable is removed from the configuration file and the file is pushed, i
 
 When pulling configuration, use `--store-command=refs` flag to restore the configuration as Parameter Store references that can be added to the secrets section of a Task Definition.
 
+### Using cStore CLI with AWS Parameter Store
```

Comment:
To be consistent with the other headers we could say, `Providing AWS Parameter Store Access`. Specifying CLI here may be too specific. This would be the policy used for direct access or CLI access.